app/events/audit_event.rb

```diff
@@ -6,7 +6,7 @@ def self.publish(audit)
     create(
       eventable: audit,
       organization_id: audit.organization_id,
-      event_time: Time.zone.now,
+      event_time: audit.updated_at,
```

Comment:
Yeah -- this one uses `updated_at` and others use `created_at` for the eventable it's tracking, how did you intend which is which?

Reply:
Because audits have a state (they have to be finalized) _and_ they can't be edited, so updated\_at is the correct thing. For the others, they can be edited but they are always "valid", so created\_at is correct.